pkg/api/meta/interfaces.go

```diff
@@ -142,4 +142,6 @@ type RESTMapping struct {
 type RESTMapper interface {
 	VersionAndKindForResource(resource string) (defaultVersion, kind string, err error)
 	RESTMapping(kind string, versions ...string) (*RESTMapping, error)
+	AddResourceAlias(alias string, resources ...string)
```

Comment:
I need access to the aliasToResource map used by `AliasesForResource` and since I cannot setup the alias on the spot, I can only see two ways of doing this: 1) export the map which is ugly or 2) export a setter that edits the map.